docs/quick_tour.rst

```diff
@@ -827,7 +819,7 @@ Pyramid and SQLAlchemy are great friends. That friendship includes a scaffold!
 
   $ pcreate --scaffold alchemy sqla_demo
   $ cd sqla_demo
-  $ python setup.py develop
+  $ env/bin/pip install -e .
```

Comment:
This example and several before it are using `env/bin/pip` whereas the examples you used before were not in a virtualenv (`python setup.py develop` not `env/bin/python setup.py develop`). I'm just adding this note to make sure that change is intentional.